test/models/export/ProductDraftWrapperTest.java

```diff
@@ -0,0 +1,53 @@
+package models.export;
+
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.ProductDraft;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.taxcategories.TaxCategory;
+import org.junit.Before;
+import org.junit.Test;
+import utils.JsonUtils;
+
+import java.util.Locale;
+import java.util.UUID;
+
+import static org.assertj.core.api.StrictAssertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ProductDraftWrapperTest {
+
+    private static final String PRODUCT_JSON_RESOURCE = "data/product-draft.json";
+
+    private ProductType productType;
+    private TaxCategory taxCategory;
+
+    @Before
+    public void setUp() {
+        productType = mock(ProductType.class);
+        final String productTypeId = UUID.randomUUID().toString();
+        when(productType.toReference()).thenReturn(Reference.of("product-type", productTypeId, productType));
+        when(productType.getId()).thenReturn(productTypeId);
```

Comment:
When you need to use mocks for your own code then this is a sign that sth. is not right with your API.

see https://github.com/commercetools/sphere-donut/pull/40/files#r41992292